src/components/views/ViewDataTable/ViewDataTableSorting.tsx

```diff
@@ -0,0 +1,161 @@
+                                            name="column"
+                                            onChange={ (evt) => handleChange({ ...evt.target, field: item.field }) }
+                                            value={ item.field }>
+                                            { gridColumns.map(gridColumn => (
+                                                <MenuItem key={ gridColumn.field }
+                                                    disabled={ sortModel.map(s => s.field).includes(gridColumn.field) }
+                                                    value={ gridColumn.field }>
+                                                    { gridColumn.headerName }
+                                                </MenuItem>
+                                            )) }
+                                        </Select>
+                                    </FormControl>
+                                </Box>
+                                <Box flex={ 1 }>
+                                    <FormControl fullWidth>
+                                        <Select
+                                            name="direction"
+                                            onChange={ (evt) => handleChange({ ...evt.target, field: item.field }) }
+                                            value={ item.sort }>
+                                            <MenuItem value="asc">Ascending</MenuItem>
+                                            <MenuItem value="desc">Descending</MenuItem>
+                                        </Select>
+                                    </FormControl>
+                                </Box>
+                                <IconButton
+                                    className={ classes.deleteButton }
+                                    data-testid="deleteSortModelItem"
+                                    onClick={ () => handleChange({ field: item.field, name: 'delete', value: true }) }>
+                                    <Delete />
+                                </IconButton>
+                            </Box>)) }
+                    </Box>
+                    <Box>
+                        <Button color="primary"
+                            disabled={ sortModel.some(item => !item.sort) }
+                            onClick={ handleAdd }
+                            size="small"
+                            startIcon={ <Add /> }
+                            variant="text">
+                            <FormattedMessage id="misc.views.viewTableSort.addButton" />
+                        </Button>
+                    </Box>
+                    <Typography variant="caption">
+                        <FormattedMessage id="misc.views.viewTableSort.hintStart" />
+                        <ShiftKeyIcon size={ 40 } svgProps={{ className: classes.shiftIcon }} />
+                        <FormattedMessage id="misc.views.viewTableSort.hintEnd" />
```

Comment:
These should all be one message, because formatting may be more complex in another language.

You can pass a React element as a value to a formatted message, i.e.:

```tsx
<FormattedMessage id="some.message.id" values={{ hintIcon: <ShiftKeyIcon size={ 40 }/> }} />
```

In your `some.message.id` message you can now do something like `press shift ({hintIcon}) to do whatever`.